akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -314,7 +314,7 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
    * a certain size.
    */
   def registerOnMemberUp[T](code: ⇒ T): Unit =
```

Comment:
I don't know, I guess this comes from ActorSystem.registerOnTermination.
Let's leave it as is.